src/internal_events/dnstap.rs

```diff
@@ -4,38 +4,75 @@ use metrics::counter;
 use vector_core::internal_event::InternalEvent;
```

Comment:
Remember to remove the `skip check-events` when you update these files for the component spec.